373-find-k-pairs-with-smallest-sums/step1.py

```diff
@@ -0,0 +1,46 @@
+from heapq import heappop, heappush
+
+
+class Solution:
+    def kSmallestPairs(
+        self, nums1: list[int], nums2: list[int], k: int
+    ) -> list[list[int]]:
+        sum_and_indices = [(nums1[0] + nums2[0], 0, 0)]
+        pushed_pairs = set((0, 0))
```

Comment:
そういえばこの書き方は正しくありませんでした。
この書き方だと「(0, 0)という2つの要素からなるiterable」として処理されてしまい、整数0のみからなる集合`{0}`となってしまいます。
https://docs.python.org/ja/3/library/stdtypes.html#set

正しくは以下のように書くべきでした。

```suggestion
        pushed_pairs = {(0, 0)}
```